### ISSUE TITLE
[question] encrypted version possible?

### ISSUE THREAD

**Issue by radu022003** [koenvervloesem/BTHome-Motion-Detection-in-CircuitPython#1]
Hello, I have searched quite long for a micropython based solution for BTHome encryption, but none found. I guess mostly because of the aes ccm mode, missing.
I would gladly help out.

**Comment by koenvervloesem** [comment]
I agree that it would be an interesting project. Apparently CircuitPython has an [aes](https://docs.circuitpython.org/en/latest/shared-bindings/aesio/index.html) module, but no CCM...
